docs/snyk.md

```diff
@@ -0,0 +1,104 @@
+# How to setup SNYK Autorun on each PR
+Readme to document how to setup SNYK for a given repository
+
+## Give SNYK access to the repo
+ * Create/login to a SNYK account
+   * https://app.snyk.io
+   * [Login] -> GitHub (or other option if relevant)
+   * This account must be the repo owner
+ * Add relevant project
```

Comment:
I found this step a little bit confusing.

When you don't have any project added to Snyk, you have to go under 'Projects' tab and select 'Add GitHub repositories to Snyk', then grant access to you public or your public and private repositories. After that, you need to select the repositories you want to add to Snyk.

So I would put something like:

If you want to add your first repository to Snyk:
     - Select 'Projects' tab
     - Select 'Add GitHub repositories to Snyk'
     - Grant the required accesses
     - Select the repository you want to add to Snyk
     - Wait for the import confirmation
If you want to add another repository:
     - From the dashboard page, select 'Add more projects'
     - Select the repository you want to add to Snyk
     - Wait for the import confirmation

Reply:
clarified in latest commit